metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -814,41 +837,95 @@ def get_refinement_from_model_includes(self, view_name: str) -> List[dict]:
                 continue
 
             refined_views.extend(
-                self.get_refinement_from_looker_view_file(
-                    included_looker_viewfile, view_name
+                self.get_refinement_from_view_list(
+                    included_looker_viewfile.views, view_name
                 )
             )
 
         return refined_views
 
+    def skip_processing(self, raw_view_name: str) -> bool:
+        if LookerRefinementResolver.is_refinement(raw_view_name):
+            return True
+
+        if self.source_config.process_refinements is False:
+            return True
+
+        return False
+
     def apply_view_refinement(self, raw_view: dict) -> dict:
         """
         Looker process the lkml file in include order and merge the all refinement to original view.
         """
-        raw_view_name: str = raw_view["name"]
+        assert raw_view.get(LookerRefinementResolver.NAME) is not None
 
-        if LookerRefinementResolver.is_refinement(raw_view_name):
-            # It is refinement no need to process
-            return raw_view
+        raw_view_name: str = raw_view[LookerRefinementResolver.NAME]
 
-        if self.source_config.process_refinements is False:
+        if self.skip_processing(raw_view_name):
             return raw_view
 
-        if raw_view_name in self.refinement_cache:
+        if raw_view_name in self.view_refinement_cache:
             logger.debug(f"Returning applied refined view {raw_view_name} from cache")
-            return self.refinement_cache[raw_view_name]
+            return self.view_refinement_cache[raw_view_name]
 
         logger.debug(f"Processing refinement for view {raw_view_name}")
 
         refinement_views: List[dict] = self.get_refinement_from_model_includes(
             raw_view_name
         )
 
-        self.refinement_cache[raw_view_name] = self.merge_refinement(
+        self.view_refinement_cache[raw_view_name] = self.merge_refinements(
             raw_view, refinement_views
         )
 
-        return self.refinement_cache[raw_view_name]
+        return self.view_refinement_cache[raw_view_name]
+
+    @staticmethod
+    def add_extended_explore(raw_view: dict, refinement_views: List[Dict]) -> None:
```

Comment:
the parameters here are explores right? can we update the names to be raw_explore and refinement_explores?

Reply:
done